### DISCUSSION TITLE
Unable to push

### DISCUSSION THREAD

**Discussion by A-PRIVATE1** [community/community#140525] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I may not open ticket as everything guarded now by AI and topic is unclear - github has all topics except when there is a problem with github.
Tried to create a copy of filezilla to start work with which is 43 mbytes whole repo.
But got:
git push -u origin main Enumerating objects: 1540, done. Counting objects: 100% (1540/1540), done. Delta compression using up to 4 threads Compressing objects: 100% (1532/1532), done. error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400 fatal: the remote end hung up unexpectedly Writing objects: 100% (1540/1540), 5.76 MiB | 4.34 MiB/s, done. Total 1540 (delta 146), reused 0 (delta 0) fatal: the remote end hung up unexpectedly Everything up-to-date

**Comment by Ali-Barzegari-d** [comment, marked as answer]
You're getting a 400 error while pushing. Here are a few quick fixes:

Check Size Limits: Make sure no files exceed 100 MB and the repo isn’t over 1 GB.
Increase Git Buffer: Run:
git config http.postBuffer 524288000
Check Internet Connection: Ensure it's stable.
Use SSH: Switch from HTTPS to SSH:
git remote set-url origin git@github.com:username/repo.git
Push Smaller Commits: Break up larger commits into smaller ones.
If you still face issues, consider contacting GitHub support.

if it does, mark it as the answer! 😪

**Reply by A-PRIVATE1** [reply]
thank you ali
